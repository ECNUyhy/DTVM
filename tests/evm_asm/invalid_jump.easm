PUSH1 0x09
JUMP
PC
JUMPDEST
