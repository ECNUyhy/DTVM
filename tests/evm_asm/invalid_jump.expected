status: bad jump destination
error_code: 8
stack: []
memory: ''
storage: {}
transient_storage: {}
return: ''
events: []
